README.md

```diff
@@ -0,0 +1,149 @@
+# M1M3 Support Software
+
+Contains realtime software which runs on NI cRIO for M1M3 suppport.
+[FPGA](https://github.com/lsst-ts/ts_m1m3supportFPGA) handling low-level
+communication, and 
+[LabView EUI (GUI panel)](https://github.com/lsst-ts/ts_m1m3supportEUI) running
+on host PC are separate projects.
+
+## Theory of operation
+
+M1M3 Support System (M1M3SS) operates Dual Axis Actuators (DAAs) and Single
+Axis Actuators (SAAs) to keep mirror in neutral buoyancy state, counteracting
+primary gravity force on the mirror. It uses hardpoints to steer the mirror,
+primary when it's raised on controlled lowered. Hardpoints are also used to
+make mirror stiff, removing wind induced oscillations.
+
+There are secondary goals operating DAAs and SAAs:
+
+* offload local hardpoints steering forces
+* apply small forces to keep mirror in optically ideal shape
+
+DAAs and SAAs are also equipped with booster valves. Those are activated to
+counteract forces experiented during telescope slews.
```

Comment:
experienced ?

Reply:
Fixed. Plus supp**p**ort and explain ILC